spec/models/file_depot_ftp_anonymous_spec.rb

```diff
@@ -0,0 +1,6 @@
+describe FileDepotFtpAnonymous do
+  it "should require credentials for anonymous" do
+    expect(FileDepotFtpAnonymous.requires_credentials?).to eq true
+    expect(FileDepotFtpAnonymous.new.login_credentials[0]).to eq "anonymous"
```

Comment:
In the future, this should probably match the entire array instead of the first element.